src/common/eeprom.c

```diff
@@ -166,11 +270,33 @@ void eeprom_dump(void) {
     }
 }
 
+void eeprom_get_hostname(char * dest){
+    char hostname_str[LAN_HOSTNAME_MAX_LEN + 1];
+    for(uint8_t i = 0; i < LAN_HOSTNAME_MAX_LEN; i++){
+        hostname_str[i] = (char)eeprom_get_var(EEVAR_LAN_HOSTNAME_0 + i).ui8;
+    }
+    hostname_str[LAN_HOSTNAME_MAX_LEN] = '\0';
```

Comment:
is this an extra measure to end the char string in case "dest" is not null terminated?

Reply:
Yes